doc/div-alg.xml

```diff
@@ -152,6 +151,14 @@ gap> SchurIndexByCharacter(GaussianRationals,G,Irr(G)[i]);
 2
 gap> SchurIndexByCharacter(CF(3),G,i);
 1
+gap> G:=AtlasGroup("J2");
```

Comment:
if you add a manual example, you need to run `gap makedoc.g` and commit file(s) in `tst` with extracted examples, to ensure that they will be tested (see 2nd section at https://github.com/gap-packages/wedderga/wiki).